packages/connectivity/README.md

```diff
@@ -60,17 +60,39 @@ var wifiIP = await (Connectivity().getWifiIP());network
 var wifiName = await (Connectivity().getWifiName());wifi network
 ```
 
-### Known Issues
+### iOS 12
 
-#### iOS 13
+To use `.getWifiBSSID()` and `.getWifiName()` on iOS >= 12, the `Access WiFi information capability` in XCode must be enabled. Otherwise, both method will return null.
```

Comment:
```suggestion
To use `.getWifiBSSID()` and `.getWifiName()` on iOS >= 12, the `Access WiFi information capability` in XCode must be enabled. Otherwise, both methods will return null.
```